actfw_core/capture.py

```diff
@@ -107,13 +120,15 @@ def cmp(config):
         else:
             fmt = self.video.set_format(config, width, height, expected_format=expected_format)
         self.capture_width, self.capture_height, self.capture_format = fmt
+        # TODO: v3.0.0. Comment out.
+        # assert type(self.capture_format) is V4L2_PIX_FMT
```

Comment:
IIUC, this assertion is correct, but not very confident. It is safe to postpone commenting out this to 3.0.0.

@notogawa What do you think?